packages/framework/src/types/step.types.ts

```diff
@@ -1,6 +1,6 @@
 import { ActionStepEnum, ChannelStepEnum } from '../constants';
-import { digestRegularOutputSchema, digestTimedOutputSchema } from '../schemas';
 import { actionStepSchemas } from '../schemas/steps/actions';
+import { digestRegularOutputSchema, digestTimedOutputSchema } from '../schemas/steps/actions/digest.schema';
```

Comment:
Is there a particular reason for this change?

Reply:
i needed to expose those in the framework, before they were not available